spyder/plugins/ipythonconsole/widgets/main_widget.py

```diff
@@ -2366,6 +2372,18 @@ def run_script(self, filename, wdir, args, debug, post_mortem,
             )
 
     # ---- For working directory and path management
+    @property
+    def current_working_directory(self):
```

Comment:
This maybe should be `get_current_working_directory` and remove the property so it will remain as function